### PR TITLE
Raise CPU limit to 150%

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/16803

### CODE REVIEW

**Comment by QuentinBisson** [comment]
This could be useful but this might trigger a few heartbeats :D

**Review by QuentinBisson — APPROVED** [review]

File: service/controller/resource/prometheus/resource.go [hunk 1]
```diff
@@ -177,8 +177,8 @@ func toPrometheus(v interface{}, config Config) (metav1.Object, error) {
 					corev1.ResourceMemory: *key.PrometheusDefaultMemory(),
 				},
 				Limits: corev1.ResourceList{
-					// cpu: 100m
-					corev1.ResourceCPU: *key.PrometheusDefaultCPU(),
+					// cpu: 150m
+					corev1.ResourceCPU: *key.PrometheusCPULimit(),
```
Comment (JosephSalisbury) [inline service/controller/resource/prometheus/resource.go:181]: is this the _default_ CPU limit? updating the naming might be helpful

Comment (QuentinBisson) [reply service/controller/resource/prometheus/resource.go:181]: I think this would make sense

Comment (TheoBrigitte) [reply service/controller/resource/prometheus/resource.go]: renamed to `PrometheusDefaultCPULimit`

**Pull request merged**